lambda/data.tf

```diff
@@ -18,4 +18,24 @@ data "aws_ssm_parameter" "staging_account_number" {
 data "aws_ssm_parameter" "prod_account_number" {
   count = var.project == "tdr" && local.environment == "mgmt" ? 1 : 0
   name  = "/mgmt/prod_account"
-}
\ No newline at end of file
+}
```

Comment:
Add line break

Reply:
I made a real effort with the line breaks this time, knew I'd miss one.

Reply:
;)